ert_shared/storage/http_server.py

```diff
@@ -110,6 +113,34 @@ def data(self, data_id):
             else:
                 return str(data)
 
+    def get_observation(self, name):
+        with StorageApi(
+            rdb_url=self._rdb_url, blob_url=self._blob_url
+        ) as api:
+            return api.observation(name)
+
+    def get_observation_tags(self, name):
+        with StorageApi(
+            rdb_url=self._rdb_url, blob_url=self._blob_url
+        ) as api:
+            try:
+                return {"tags": api.observation_tags(name)}
```

Comment:
I think it is a good idea we move all generation of return schema to the storage_api, unless it directly relates to http stuff (see resolve_urls)